pages/treeherder.py

```diff
@@ -99,6 +117,18 @@ def filter_by(self, term):
     def filter_unclassified_jobs(self):
         self.find_element(*self._unclassified_failure_filter_locator).click()
 
+    def get_next_ten_results(self):
+        self.find_element(*self._next_ten_locator).click()
+        self.wait.until(lambda s: len(self.result_sets) == 20)
+
+    def get_next_twenty_results(self):
+        self.find_element(*self._next_twenty_locator).click()
+        self.wait.until(lambda s: len(self.result_sets) == 30)
+
+    def get_next_fifty_results(self):
```

Comment:
So these aren't alphabetical, but in this case I prefer how you've sorted them by value.